doc/sphinx/practical-tools/utilities.rst

```diff
@@ -74,6 +74,10 @@ to literally pass an argument ``foo`` to ``coqc``: in the
 example, this allows to pass the two-word option ``-w all`` (see
 :ref:`command line options <command-line-options>`).
 
+Note that, in general, it is mandatory to specify a ``-R/-Q`` flag for
+your project, failing to do will produce files without a proper
+library prefix, which are to be used only in special cases.
```

Comment:
```suggestion
Normally you should specify a ``-R/-Q`` flag for
your project.  Omitting it will generate compiled files without a proper
library prefix, which is for use in special cases.
```

Who would use the "special cases" and why?

Reply:
Should is not OK, it is indeed mandatory. Special cases means "I am a developer of Coq and know what I am doing"

Reply:
Actually seems like a good idea to make `coqc foo.v` fail if there is no scope, but the compat impact is huge here.